core/src/validator.rs

```diff
@@ -570,25 +566,25 @@ impl Validator {
 
         info!("Cleaning accounts paths..");
         *start_progress.write().unwrap() = ValidatorStartProgress::CleaningAccounts;
-        let mut start = Measure::start("clean_accounts_paths");
+        let mut timer = Measure::start("clean_accounts_paths");
         cleanup_accounts_paths(config);
-        start.stop();
-        info!("done. {}", start);
+        timer.stop();
+        info!("Cleaning accounts paths done. {timer}");
 
         snapshot_utils::purge_incomplete_bank_snapshots(&config.snapshot_config.bank_snapshots_dir);
         snapshot_utils::purge_old_bank_snapshots_at_startup(
             &config.snapshot_config.bank_snapshots_dir,
         );
 
         info!("Cleaning orphaned account snapshot directories..");
-        if let Err(e) = clean_orphaned_account_snapshot_dirs(
+        let mut timer = Measure::start("clean_orphaned_account_snapshot_dirs");
+        clean_orphaned_account_snapshot_dirs(
             &config.snapshot_config.bank_snapshots_dir,
             &config.account_snapshot_paths,
-        ) {
-            return Err(format!(
-                "Failed to clean orphaned account snapshot directories: {e:?}"
-            ));
-        }
+        )
+        .map_err(|err| format!("Failed to clean orphaned account snapshot directories: {err:?}"))?;
```

Comment:
does `fmt` not want this indented? :thinking:

Reply:
🤷